cli/cli.go

```diff
@@ -224,7 +224,7 @@ func cmdWithAuth0Arg(cfg *Config, cmd func(*HttpContext)) func(c *cli.Context) e
 func Priam(args []string, defaultCfgFile string, infoW, errorW io.Writer) {
 	var err error
 	cfg := &Config{}
-	cli.HelpFlag.Usage = "show help for given command or subcommand"
```

Comment:
can you add a comment to explain why you needed to comment this? (and not removing it instead?)

Reply:
removed it.